__docs__/wonder-blocks-labeled-field/labeled-field.stories.tsx

```diff
@@ -0,0 +1,165 @@
+    const [singleSelectValue, setSingleSelectValue] = React.useState("");
+    const [multiSelectValue, setMultiSelectValue] = React.useState<string[]>(
+        [],
+    );
+    const [searchValue, setSearchValue] = React.useState("");
+
+    return (
+        <View style={{gap: spacing.large_24}}>
+            <LabeledField
+                {...args}
+                label="Text Field"
+                field={
+                    <TextField
+                        value={textFieldValue}
+                        onChange={setTextFieldValue}
+                    />
+                }
+            />
+            <LabeledField
+                {...args}
+                label="Text Area"
+                field={
+                    <TextArea
+                        value={textAreaValue}
+                        onChange={setTextAreaValue}
+                    />
+                }
+            />
+
+            <LabeledField
+                {...args}
+                label="Single Select"
+                field={
+                    <SingleSelect
+                        placeholder="Choose a fruit"
+                        selectedValue={singleSelectValue}
+                        onChange={setSingleSelectValue}
+                        error={!!args.error}
+                    >
+                        <OptionItem label="Mango" value="mango" />
+                        <OptionItem label="Strawberry" value="strawberry" />
+                        <OptionItem label="Banana" value="banana" />
+                    </SingleSelect>
+                }
+            />
+
+            <LabeledField
+                {...args}
+                label="Multi Select"
+                field={
+                    <MultiSelect
+                        selectedValues={multiSelectValue}
+                        onChange={setMultiSelectValue}
+                        error={!!args.error}
+                    >
+                        <OptionItem label="Mango" value="mango" />
+                        <OptionItem label="Strawberry" value="strawberry" />
+                        <OptionItem label="Banana" value="banana" />
+                    </MultiSelect>
+                }
+            />
+
+            <LabeledField
+                {...args}
+                label="Search"
+                field={
+                    <SearchField
+                        value={searchValue}
+                        onChange={setSearchValue}
+                    />
+                }
+            />
+        </View>
+    );
+};
+
+/**
+ * The `LabeledField` component can be used with form field components such as:
+ * - `TextField`
+ * - `TextArea`
+ * - `SingleSelect`
+ * - `MultiSelect`
+ * - `SearchField`
+ */
+export const Fields: StoryComponentType = {
+    args: {
+        description: "Helpful description text",
+        required: true,
+    },
+    render: AllFields,
+};
+
+export const Error: StoryComponentType = {
```

Comment:
**suggestion:** It'd be nice adding a description here (it's fine if you were planning to add it in a future PR).

Reply:
Will add to it in a future PR that addresses error state and validation!